transmart-api-server/build.gradle

```diff
@@ -36,3 +36,19 @@ dependencies {
 bootRun {
     jvmArgs = ['-Dspring.output.ansi.enabled=always']
 }
+
+task executableWar(type: Copy) {
+    from "${libsDir}/${project.name}-${project.version}.war"
```

Comment:
Do we really need this extra task where we copy the war file to another directory?